api/client/system/info.go

```diff
@@ -214,3 +232,14 @@ func runInfo(dockerCli *client.DockerCli) error {
 	}
 	return nil
 }
+
+func formatInfo(dockerCli *client.DockerCli, info types.Info, format string) error {
+	tmpl, err := templates.Parse(format)
+	if err != nil {
+		return cli.StatusError{StatusCode: 64,
```

Comment:
It is just copied from here :sweat_smile: https://github.com/docker/docker/pull/14194/files#diff-1d801b8b2dcffcde77b5995194bb3171R56